### PR TITLE
Revert to apache

### PR DESCRIPTION
**Pull request by thespad**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-smokeping/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description:
Latency with nginx/fcgiwrap is unacceptably high and so far I haven't found a way to improve it.

### CODE REVIEW

File: root/etc/s6-overlay/s6-rc.d/init-smokeping-config/run [hunk 1]
```diff
@@ -13,20 +15,34 @@ PREV_DIR=$(pwd)
 cd /defaults/smoke-conf || exit
 shopt -s globstar nullglob
 for i in *; do
-    if [[ ! -e "/config/${i}" ]]; then
-        cp "${i}" "/config/${i}"
-    fi
+    [[ ! -e "/config/${i}" ]] && cp -v "${i}" "/config/${i}"
```
Comment (nemchik) [inline root/etc/s6-overlay/s6-rc.d/init-smokeping-config/run:18]: Use full `if` syntax.

**Review by nemchik — CHANGES_REQUESTED** [review]

File: root/etc/s6-overlay/s6-rc.d/init-smokeping-config/run (hunk 1 above)
Comment (thespad) [reply root/etc/s6-overlay/s6-rc.d/init-smokeping-config/run:18]: Bugger missed that in the merge.

Comment (thespad) [reply root/etc/s6-overlay/s6-rc.d/init-smokeping-config/run]: Resolved in [94d2c12](https://github.com/linuxserver/docker-smokeping/pull/141/commits/94d2c1250280cead3805f4b1049b73db307b2cb6)

**Comment by nemchik** [comment]
Just for fun, I believe this is a comparison of this PR against the repo prior to #136 
https://github.com/linuxserver/docker-smokeping/compare/9d5800eb4e7db5551d1c6d6d52ecbf7548928d66...94d2c1250280cead3805f4b1049b73db307b2cb6

**Comment by nemchik** [comment]
Arm (32 and 64) CI report looks broken.

**Comment by thespad** [comment]
> Arm (32 and 64) CI report looks broken.

Looks like a qemu issue, aarch64 runs fine natively.

**Review by nemchik — APPROVED** [review]

**Pull request merged**